### PR TITLE
Various improvements with the new bug type

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

**Comment by sylvestre** [comment]
I kept bug 1204123 open for testing

**Comment by marco-c** [comment]
The "meta" case is not too bad (it's possible that a meta bug is a big defect and there are many steps to fix it, filed as separate bugs)

**Comment by sylvestre** [comment]
Yeah, i agree

File: auto_nag/scripts/feature_but_type_defect_task.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+class RegressionButDefectTask(BzCleaner):
+    def __init__(self):
+        super(RegressionButDefectTask, self).__init__()
+
+    def description(self):
+        return 'Get defect or task with the "feature" keyword'
+
+    def name(self):
+        return "feature_but_type_defect_task"
```
Comment (calixteman) [inline auto_nag/scripts/feature_but_type_defect_task.py]: crypto nit: the strings delimiters should be consistent over the same file

File: templates/feature_but_type_defect_task.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<p>The following {{ plural('bug has', data, pword='bugs have') }} the 'feature' keyword but are task or defect:
```
Comment (calixteman) [inline templates/feature_but_type_defect_task.html]: nit: use plural for "are"

File: templates/regression_but_type_enhancement_task.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<p>The following {{ plural('bug has', data, pword='bugs have') }} the 'regression' keyword but are enhancement or task:
```
Comment (calixteman) [inline templates/regression_but_type_enhancement_task.html]: plural for "are" too

**Pull request merged**